sw/device/tests/penetrationtests/firmware/BUILD

```diff
@@ -156,3 +156,37 @@ opentitan_test(
     ),
     deps = FIRMWARE_DEPS_SCA,
 )
+
+manifest(d = {
+    "name": "manifest",
+    "identifier": hex(CONST.ROM_EXT),
+    "version_major": ROM_EXT_VERSION.MAJOR,
+    "version_minor": ROM_EXT_VERSION.MINOR,
+    "security_version": ROM_EXT_VERSION.SECURITY,
+    "visibility": ["//visibility:public"],
+})
+
+offline_presigning_artifacts(
+    name = "chip_pen_test_sca_presigning",
+    testonly = True,
+    srcs = [
+        ":chip_pen_test_sca_silicon_owner_a2_rom_ext",
+    ],
+    # TODO: change to real keys
+    ecdsa_key = {
+        "//sw/device/silicon_creator/rom/keys/fake/ecdsa:test_key_0_ecdsa_p256": "fake_test_key_0",
+    },
+    spx_key = {
+        "//sw/device/silicon_creator/rom/keys/fake/spx:test_key_0_spx": "test_key_0_spx_key",
+    },
```

Comment:
And these should point at your `keyset`s for your application keys.

Since your application keys are in the `ot-provisioning-google-skus` repo, this infra might be better located there.